src/source/generator.scala

```diff
@@ -185,7 +196,9 @@ package object generatorTools {
       "fooBar!" -> camelLowerStrict,
       "foo_bar!" -> underLowerStrict,
       "Foo_Bar!" -> underUpperStrict,
-      "FOO_BAR!" -> underCaps
+      "FOO_BAR!" -> underCaps,
```

Comment:
do we need to update line 186 as well?

Reply:
Yeah, that sounds fair.
My idea was to keep the default as it is and apply `camelUpperID` via command line option but given this is our own patch, it makes sense to change the defaults as well.